src/main/java/com/google/sps/utils/Game.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.utils;
+import com.google.appengine.api.datastore.Entity;
+import java.lang.IllegalArgumentException;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+ 
+public final class Game {
+    DatastoreService datastore;
+    private Entity entity;
+ 
+    public Game() {
+        datastore = DatastoreServiceFactory.getDatastoreService();
+        entity = new Entity("Game");
+    }
+ 
+    public void setGameId(String gameId) {
+        entity.setProperty("gameId", gameId);
+        datastore.put(entity);
+    }
+ 
+    public void setGameName(String game_name) {
+        entity.setProperty("gameName", game_name);
+        datastore.put(entity);
+    }
+ 
+    public void setQuizQuestion(String quiz_question) {
+        entity.setProperty("quizQuestion", quiz_question);
+        datastore.put(entity);
+    }
+ 
+    public void setQuizTimestamp(long quiz_timestamp) {
```

Comment:
you might want to add a short description of each of these properties

Reply:
got it!